lib/prepare/resolveOptions.js

```diff
@@ -106,6 +106,12 @@ module.exports = async function resolveOptions (sourceDir) {
   const markdown = createMarkdown(siteConfig)
 
   // resolve pageFiles
+  const files = ['**/*.md', '!.vuepress', '!node_modules']
+  if (siteConfig.dest) {
+    // exclude dest folder from pageFiles
+    const outDirRelative = path.relative(sourceDir, outDir)
+    files.push('!' + siteConfig.dest)
```

Comment:
You just created a new variable, but where do you use it?

Reply:
Sorry about that, fixed now.